sdk/eventhub/Azure.Messaging.EventHubs/tests/Consumer/LastEnqueuedEventPropertiesTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.EventHubs.Consumer;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="LastEnqueuedEventProperties" />
+    ///   struct.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class LastEnqueuedEventPropertiesTests
+    {
+        /// <summary>
+        ///   Verifies functionality of the <see cref="LastEnqueuedEventProperties "/>
+        ///   equality.
+        /// </summary>
+        ///
+        [Test]
+        public void AnInstanceIsEqualToItself()
+        {
+            var first = new LastEnqueuedEventProperties(lastSequenceNumber: 123, lastOffset: 887, lastEnqueuedTime: DateTimeOffset.Parse("2015-10-27T12:00:00Z"), lastReceivedTime: DateTimeOffset.UtcNow);
+            var second = first;
```

Comment:
This feels like a bit of a weird test because `LastEnqueuedEventProperties` is a value type, especially given the next test we have.

Reply:
Agreed.  Pretty sure that I copy/pasted that from a reference type set of equality tests;  doesn't make sense here.